### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by zeroregard**
- Move `font_awesome_flutter` to live inside project but add to `.gitignore`
- More information on the setup screen - I thought it wasn't too clear what was the point when I hit 'next' so I just added some more text.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -49,3 +49,6 @@ app.*.map.json
 .vscode/
 linux/flutter/generated_plugins.cmake
 windows/flutter/generated_plugins.cmake
+
+# Private/paid dependencies
+/dependencies/font_awesome_flutter
```
Comment (zeroregard) [inline .gitignore:54]: I didn't get why this one had to be installed manually, is it not just this that's available?
https://pub.dev/packages/font_awesome_flutter/versions

File: lib/pages/landing_page/firsttime_setup.dart [hunk 2]
```diff
@@ -52,7 +52,7 @@ class FirstTimeSetup extends StatelessWidget {
       elevation: elevation,
       child: Container(
         width: Nav().screenWidth - (Nav().screenWidth / 5),
-        height: Nav().screenHeight - (Nav().screenHeight / 1.5),
+        height: 320,
```
Comment (zeroregard) [inline lib/pages/landing_page/firsttime_setup.dart:55]: Probably we need to do some tweaking on this screen at some point.
I didn't catch why the height and width are defined this way, probably this component should scale to its children content?
I remember this being a tricky one in flutter though :D

File: .gitignore (hunk 1 above)
Comment (zeroregard) [reply .gitignore:54]: Nvm, right, it's the pro version!

**Review by VladTheSheep — APPROVED** [review]

File: lib/pages/landing_page/firsttime_setup.dart (hunk 2 above)
Comment (VladTheSheep) [reply lib/pages/landing_page/firsttime_setup.dart:55]: Best to leave at a fixed height perhaps? So the height of the widget won't change abruptly once it moves to the "Loading" widget.

**Pull request merged**